source/Government.cpp

```diff
@@ -123,33 +165,18 @@ void Government::Load(const DataNode &node)
 			}
 		}
 		else if(key == "penalty for")
-		{
+			PenaltyHelper(child, penaltyFor);
+		else if(key == "custom penalties")
 			for(const DataNode &grand : child)
-				if(grand.Size() >= 2)
+			{
```

Comment:
pretty sure u dont need this set of braces
can keep it if you want though

Reply:
They don't seem to be necessary, but I think it looks better in VSCode with them, and it doesn't seem to be against the style guide.